README.md

```diff
@@ -194,14 +195,14 @@ GET /v1/properties/:propertyReference/
 
 ### Response
 
+This should mostly mirror the individual results from the List properties by
+ID endpoint.
+
 ```json
 {
-  "results": [
-    {
-      //TBC
-      "postcode": "AB1 1AA"
-    }
-  ]
+  "propertyReference": "00032896"
+  "postcode": "AB1 1AA"
```

Comment:
Don't you need to comma separate these key-value pairs?

Reply:
Oups!

Reply:
@leeky I've amended the commit

Reply:
:+1: